charts/tidb-backup/templates/scripts/_start_restore.sh.tpl

```diff
@@ -9,15 +9,23 @@ host=`echo {{ .Values.clusterName }}_TIDB_SERVICE_HOST | tr '[a-z]' '[A-Z]' | tr
 downloader \
   --cloud=gcp \
   --bucket={{ .Values.restore.gcp.bucket }} \
-  --srcDir={{ .Values.restore.gcp.srcDir }} \
+  --srcDir={{ .Values.restore.srcDir }} \
   --destDir=${dataDir}
+{{- end }}
+
+{{- if .Values.restore.ceph }}
```

Comment:
The current template allows for the downloader to be called twice. I think and `else` would be better.